sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -80,10 +80,8 @@ class HiveContext private[hive](
   protected[sql] override def sharedState: HiveSharedState = {
     sparkSession.sharedState.asInstanceOf[HiveSharedState]
   }
-
```

Comment:
You're correct @srowen the line was there, but the file had changed since your first review of my *minor* change so I had to rebase. While rebasing and fixing the merge conflict I noticed the two lines and remove one. Since I was fixing the merge conflict I introduced the additional change (as it should've been noticed and fixed before).